src/mumble/CMakeLists.txt

```diff
@@ -321,7 +321,7 @@ set(MUMBLE_SOURCES
 	"mumble.qrc"
 	"flags/mumble_flags_0.qrc"
 	"${CMAKE_SOURCE_DIR}/themes/DefaultTheme.qrc"
-)
+        ConnectedQuitBehaviour.cpp ConnectedQuitBehaviour.h)
```

Comment:
These should be added further up in the block for the other regular source files. Also keep one file per line, please